docs/customization.rst

```diff
@@ -153,8 +153,7 @@ Feature Extractor Level
 *Resampling the image*
 
 - ``resampledPixelSpacing`` [None]: List of 3 floats (>= 0), sets the size of the voxel in (x, y, z) plane when resampling.
-  A value of 0 is replaced with the spacing for that dimension as it is in the original (non-resampled) mask, thereby
-  enabling only in-plane resampling, for example.
+  A value of 0 is replaced with the spacing for that dimension as it is in the original (non-resampled) mask.For example, to perform only in-plane resampling, the x and y values alone should be edited (e.g.: [2,2,0]). In-plane resolution is always relative to image acquisition plane (i.e. axial, coronal or sagittal).
```

Comment:
Maybe a little nit picky, but can you add a space after the period?
`original (non-resampled) mask.For example, to`